samples/snippets/src/main/java/com/example/translate/BatchTranslateText.java

```diff
@@ -85,7 +85,7 @@ public static void batchTranslateText(
           client.batchTranslateTextAsync(request);
 
       System.out.println("Waiting for operation to complete...");
-      BatchTranslateResponse response = future.get(300, TimeUnit.SECONDS);
```

Comment:
add a random value here

Reply:
So Should I generate random between min-max timeout limit; for example,
```
random number between 300 (min)  <= random_number<= 450(max)
```